src/db/sysdb_subdomains.c

```diff
@@ -229,6 +244,26 @@ check_subdom_config_file(struct confdb_ctx *confdb,
           sd_conf_path, CONFDB_DOMAIN_FQ,
           subdomain->fqnames ? "TRUE" : "FALSE");
 
+    /* allow to set pam_gssapi_services */
+    ret = confdb_get_param(confdb, tmp_ctx, sd_conf_path,
+                           CONFDB_PAM_GSSAPI_SERVICES, &values);
+    if (ret == EOK) {
+        if (values[0] == NULL) {
+            /* Set but empty. */
```

Comment:
This unfortunately does not work `confdb_get_param()` returns `EOK` and an array with an empty first element even if the option does not exists at all and as a result breaks the inheritance.

Reply:
I workaround it by choosing "-" as default value. So you can use `pam_gssapi_services = -` to disable it for the subdomain. This is the same as providing fake service name, but hopefully more intuitive.